packages/design-system/src/components/Button/style.scss

```diff
@@ -90,6 +90,28 @@
     }
   }
 
+  &--text {
+    background-color: transparent;
```

Comment:
Loader for other buttons (even when the text is long) is centered, but for `text` button it's aligned to left:
<img width="882" alt="Monosnap 2020-06-05 17-35-12" src="https://user-images.githubusercontent.com/24607370/83895420-f4890a00-a752-11ea-90c0-7fdd9f45569e.png">

Reply:
I'll discuss it with Jo

Reply:
Discussed and fixed in cb726c9 - text buttons will behave like other buttons :-)